app/src/main/java/com/sopt/now/compose/feature/main/MainViewModel.kt

```diff
@@ -5,53 +5,65 @@ import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
-import com.sopt.now.compose.data.ServicePool
+import com.sopt.now.compose.data.remote.repository.FriendRepository
+import com.sopt.now.compose.data.remote.repository.UserRepository
 import com.sopt.now.compose.data.remote.response.FriendsResponse
 import com.sopt.now.compose.data.remote.response.UserResponse
+import dagger.hilt.android.lifecycle.HiltViewModel
 import kotlinx.coroutines.launch
+import javax.inject.Inject
 
-class MainViewModel : ViewModel() {
-    private val _userInfo = MutableLiveData<UserResponse.User>()
-    val userInfo: LiveData<UserResponse.User> = _userInfo
+@HiltViewModel
+class MainViewModel
+    @Inject
+    constructor(
+        private val userRepository: UserRepository,
+        private val friendRepository: FriendRepository,
+    ) : ViewModel() {
+        private val _userInfo = MutableLiveData<UserResponse.User>()
+        val userInfo: LiveData<UserResponse.User> = _userInfo
 
-    private val _friendsInfo = MutableLiveData<List<FriendsResponse.Data>>()
-    val friendsInfo: LiveData<List<FriendsResponse.Data>> = _friendsInfo
+        private val _friendsInfo = MutableLiveData<List<FriendsResponse.Data>>()
+        val friendsInfo: LiveData<List<FriendsResponse.Data>> = _friendsInfo
 
-    private val _userId = MutableLiveData<Int>()
-    val userId: LiveData<Int> = _userId
+        private val _userId = MutableLiveData<Int>()
+        val userId: LiveData<Int> = _userId
 
-    fun getUserInfo(userId: Int) {
-        _userId.value = userId
+        fun getUserInfo(userId: Int) {
```

Comment:
```suggestion
        fun loadUserInfo(userId: Int) {
```

getXXX 라는 네이밍을 사용하는 개발자는 무언가 반환받을것을 기대할거에요! 하지만, 실제로 해당 함수는 데이터를 load 하는 역할만 하고 있네요.

Reply:
updateUserInfo도 괜찮은 것 같네요...!